src/clientversion.cpp

```diff
@@ -99,5 +99,7 @@ std::string FormatSubVersion(const std::string& name, int nClientVersion, const
         ss << ")";
     }
     ss << "/";
+    if (!fBaseNameOnly)
+        ss << "UASF-Segwit:0.3(BIP148)/";
```

Comment:
This is not necessary, if this is merged it will be part of the normal Litecoin Core codebase, and released in 0.14.0. Please remove this and the tests below.